components/tab-switcher.jsx

```diff
@@ -0,0 +1,81 @@
+// Note: source code of this component is pretty much copied directly from https://github.com/mozilla/science.mozilla.org
```

Comment:
<img width="987" alt="screen shot 2016-10-03 at 2 45 05 pm" src="https://cloud.githubusercontent.com/assets/2896608/19055288/04a946fe-8978-11e6-9eb6-4f8bd2daffbb.png">

One of the dependencies of `mofo-ui` uses `react-dom` which isn't available in React v0.13.

@alanmoo do you have any suggestion on how to fix this error?